docs/sources/zoom/example-api-responses/sanitized/user-settings.json

```diff
@@ -0,0 +1,285 @@
+    "workplace_by_facebook":true,
+    "transfer_meetings_between_devices":true,
+    "allow_show_zoom_windows":false,
+    "meeting_summary_with_ai_companion":{
+      "enable":true,
+      "auto_enable":true,
+      "who_will_receive_summary":1
+    },
+    "ai_companion_questions":{
+      "enable":true,
+      "auto_enable":true,
+      "who_can_ask_questions":1
+    }
+  },
+  "profile":{
+    "recording_storage_location":{
+      "allowed_values":[
+        "US"
+      ],
+      "value":"US"
+    }
+  },
+  "recording":{
+    "ask_host_to_confirm_disclaimer":true,
+    "ask_participants_to_consent_disclaimer":true,
+    "auto_delete_cmr":true,
+    "auto_delete_cmr_days":90,
+    "record_files_separately":{
+      "active_speaker":true,
+      "gallery_view":true,
+      "shared_screen":true
+    },
+    "display_participant_name":true,
+    "recording_thumbnails":true,
+    "optimize_recording_for_3rd_party_video_editor":true,
+    "save_panelist_chat":true,
+    "save_poll_results":true,
+    "save_close_caption":true,
+    "auto_recording":"cloud",
+    "cloud_recording":true,
+    "host_pause_stop_recording":true,
+    "ip_address_access_control":{
+      "enable":true
+    },
+    "local_recording":true,
+    "record_audio_file":true,
+    "record_gallery_view":true,
+    "record_speaker_view":true,
+    "recording_audio_transcript":true,
+    "recording_disclaimer":true,
+    "smart_recording":{
+      "create_recording_highlights":true,
+      "create_smart_chapters":true,
+      "create_next_steps":true
+    },
+    "recording_password_requirement":{
+      "have_letter":true,
+      "have_number":true,
+      "have_special_character":true,
+      "length":10,
+      "only_allow_numeric":true
+    },
+    "save_chat_text":true,
+    "show_timestamp":true
+  },
+  "schedule_meeting":{
+    "audio_type":"telephony",
+    "embed_password_in_join_link":true,
+    "force_pmi_jbh_password":true,
+    "host_video":true,
+    "join_before_host":true,
+    "participants_video":true,
+    "personal_meeting":true,
+    "pstn_password_protected":true,
+    "require_password_for_instant_meetings":true,
+    "require_password_for_scheduled_meetings":true,
+    "require_password_for_scheduling_new_meetings":true,
+    "use_pmi_for_instant_meetings":true,
+    "use_pmi_for_scheduled_meetings":true,
+    "continuous_meeting_chat":{
+      "enable":true,
+      "can_add_external_users":true,
+      "auto_add_invited_external_users":true
+    }
+  },
+  "telephony":{
+    "show_international_numbers_link":true,
+    "telephony_regions":{
+      "allowed_values":[
+        "CNTB"
+      ],
+      "selection_values":"USTB"
+    },
+    "third_party_audio":true
+  },
+  "tsp":{
+    "call_out":true,
+    "call_out_countries":[
+      { }
+    ],
+    "show_international_numbers_link":true
+  },
+  "whiteboard":{
+    "out_meeting_advanced_whiteboard":true,
+    "in_meeting_advanced_whiteboard":true
+  }
```

Comment:
the above has a TON of stuff; should use a Json schema filter to cut down to what we need.